src/vector/geometry.rs

```diff
@@ -114,6 +114,33 @@ impl Geometry {
         };
     }
 
+    pub fn set_point_zm(&mut self, i: usize, point: (f64, f64, f64, f64)) {
```

Comment:
Given that the underlying gdal function only accepts a `c_int` (which should be 32bit on most platforms?) we might want to change the index to `u32` instead?